sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/implementation/converters/ContinuationTokenSerializer.java

```diff
@@ -0,0 +1,15 @@
+package com.azure.digitaltwins.core.implementation.converters;
+
+import com.azure.core.util.serializer.JacksonAdapter;
+import com.azure.core.util.serializer.SerializerEncoding;
+
+public class ContinuationTokenSerializer {
+    public static String serialize(String continueationToken) {
```

Comment:
XML comments? I know you need to do this but I did not understand why we need this. What are we serializing a string to and why?